packages/flutter/lib/src/cupertino/text_selection_toolbar_button.dart

```diff
@@ -0,0 +1,68 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/widgets.dart';
+
+import 'button.dart';
+import 'colors.dart';
+
+const TextStyle _kToolbarButtonFontStyle = TextStyle(
+  inherit: false,
+  fontSize: 14.0,
+  letterSpacing: -0.15,
+  fontWeight: FontWeight.w400,
+);
+
+// Colors extracted from https://developer.apple.com/design/resources/.
+// TODO(LongCatIsLooong): https://github.com/flutter/flutter/issues/41507.
+const Color _kToolbarBackgroundColor = Color(0xEB202020);
+
+// Eyeballed value.
+const EdgeInsets _kToolbarButtonPadding = EdgeInsets.symmetric(vertical: 16.0, horizontal: 18.0);
+
+/// A button in the style of the iOS text selection toolbar buttons.
+class CupertinoTextSelectionToolbarButton extends StatelessWidget {
+  /// Create an instance of [CupertinoTextSelectionToolbarButton].
+  const CupertinoTextSelectionToolbarButton({
+    Key? key,
+    this.onPressed,
+    required this.child,
+  }) : super(key: key);
+
+  /// The child of this button.
+  ///
+  /// Usually a [Text] or an [Icon].
+  final Widget child;
+
+  /// Called when this button is pressed.
+  final VoidCallback? onPressed;
```

Comment:
should we may this non-nullable? If a menu item is not available shouldn't we take it off from the menu?

Reply:
This is used for the "next" button when on the final page of a menu that overflows.  Here's a terrible screenshot of a custom menu that has it, but it also exists in the native-looking menu:

![Screen Shot 2021-01-12 at 1 51 30 PM](https://user-images.githubusercontent.com/389558/104378722-4bbb3e00-54dd-11eb-808b-8410fdfd2a7b.png)